sanity-io/sanity
# refactor(base): migrate Presence to `@sanity/ui` and styled components

### Description

Migrate presence from /base to use `@sanity/ui` and styled components

### What to review

- It's likely easier to do it if you have two page instances open in order to see your user on the presence of a document
  - I used `presence-test` -> and any of the files within

- You should also find another user in the studio and try to be in the same document
  - hover over the users to be sure that the tooltips work as expected
  - make sure that the top and bottom sticky work as expected

- Make sure to test out when the Presence is on debug 
  - go to `presence` -> `constants.ts`
  - Change `debug` constant to `true`

### Notes for release

Migrate Presence to use `@sanity/ui` and styled components
